### PR TITLE
add basic benchmark

### PR DESCRIPTION
**Pull request by josharian**
This change is [Reviewable](https://reviewable.io/reviews/tailscale/tailscale/655)

### CODE REVIEW

**Comment by josharian** [comment]
No rush on looking at this; I'm closing the lid on this week. FYI, empirically, the mean packet size that passes through derp is about 570 bytes. I have a few changes that build atop this, which I will resume next week, assuming this looks basically sane to you.

File: derp/derp_test.go [hunk 1]
```diff
@@ -742,3 +742,69 @@ func TestForwarderRegistration(t *testing.T) {
...
+func BenchmarkSendRecv(b *testing.B) {
+	for _, size := range []int{10, 100, 1000, 10000} {
+		b.Run(
+			fmt.Sprintf("msgsize=%d", size),
+			func(b *testing.B) { benchmarkSendRecvSize(b, size) },
+		)
+	}
+}
+
+var bitbucketLog = func(format string, args ...interface{}) {}
```
[the remark this answers is not in the document]
Comment (bradfitz) [reply derp/derp_test.go:755]: https://godoc.org/tailscale.com/types/logger#Discard

File: derp/derp_test.go [hunk 2]
```diff
@@ -742,3 +742,69 @@ func TestForwarderRegistration(t *testing.T) {
 		u1: testFwd(3),
 	})
 }
+
+func BenchmarkSendRecv(b *testing.B) {
+	for _, size := range []int{10, 100, 1000, 10000} {
+		b.Run(
+			fmt.Sprintf("msgsize=%d", size),
+			func(b *testing.B) { benchmarkSendRecvSize(b, size) },
```
[the remark this answers is not in the document]
Comment (josharian) [reply derp/derp_test.go:750]: I don't have a line limit; I break manually when it looks too long to me. Happy to leave this as a single line: done.

File: derp/derp_test.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (josharian) [reply derp/derp_test.go:755]: Thanks.

File: derp/derp_test.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (danderson) [reply derp/derp_test.go]: FWIW: my personal preference is no linebreaks that gofmt doesn't volunteer by itself. If that results in hard to read lines (admittedly a subjective thing), I break into multiple separate lines by declaring intermediate variables, so my long line can s/long thing/short varname/.

**Pull request merged**